vms/shift/tests.py

```diff
@@ -1,105 +1,139 @@
 from datetime import datetime
+
 from django.contrib.auth.models import User
 from django.core.exceptions import ObjectDoesNotExist
 from django.test import TestCase
+
 from event.models import Event
 from job.models import Job
-from organization.models import Organization
 from shift.models import Shift, VolunteerShift
 from shift.services import *
```

Comment:
Here I used import * because for each function from shift.services there must be a test